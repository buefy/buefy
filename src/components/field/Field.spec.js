import { shallowMount } from '@vue/test-utils'
import BField from '@components/field/Field'

describe('BField', () => {
    it('is called', () => {
        const wrapper = shallowMount(BField)
        expect(wrapper.name()).toBe('BField')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })
})
