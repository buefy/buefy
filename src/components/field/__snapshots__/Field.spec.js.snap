// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BField render correctly 1`] = `
<div class="field">
    <!---->
    <!---->
</div>
`;
